src/Plugin/Field/FieldFormatter/GoogleFormFormatter.php

```diff
@@ -38,11 +54,19 @@ public static function isApplicable(FieldDefinitionInterface $field_definition)
    * {@inheritDoc}
    */
   public function viewElements(FieldItemListInterface $items, $langcode) {
+
     $elements = [];
 
     /** @var \Drupal\Core\Entity\Plugin\DataType\EntityAdapter $parent */
     $parent = $items->getParent();
     $iframe_title = $parent->getEntity()->label();
+    $fields = $parent->getEntity()->getFields();
+    $iframe_height = '600';
+    $iframe_height_field = $fields[$this->iframeHeightField];
```

Comment:
need to check if `$fields[$this->iframeHeightField]` is set

Reply:
outdated.